docs/src/reference/workflow-config-file.rst

```diff
@@ -163,6 +166,12 @@ region
 -  type: string
 -  description: Name of the region the corresponding build belongs to (based on standard values in the ``region`` metadata field).
 
+.. warning::
+
+   The presence of a ``region`` key will result in the metadata being adjusted in potentially surprising ways.
+   For all metadata rows which are not in this region the 'location' will be removed (set to an empty string), and the 'division' and 'country' will be changed to their corresponding region.
+   Additionally, a 'focal' column will be added, with True/False values depending on if the row matches the provided region.
```

Comment:
All done, thanks!

(My understanding is that "which" and "that" can [both be used](https://www.merriam-webster.com/words-at-play/when-to-use-that-and-which) for restrictive clauses, but I have no problem choosing the one you prefer 😄 )

Reply:
I also had that understanding and found myself using "which" more often, until @huddlej made these changes https://github.com/nextstrain/ncov/pull/942#discussion_r864278649

But now I'm confused because "that" and "which" are mixed in [these steps](https://github.com/nextstrain/ncov/commit/c5fafa3eaa9c6d56957dc26058ffb1799192aac1#diff-cd2ca48fb4e7959e79f8b62db780a10b1e9540c57c2572028d236e3a92651fb4R129-R132).

I also think this is a silly detail, both options get the point across and maybe one reader will think twice about it 😅